app/services/rapid_ready_for_decision/fast_track_pdf_generator.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 module RapidReadyForDecision
-  class HypertensionPdfGenerator
+  class FastTrackPdfGenerator
```

Comment:
@allthesignals To customize for specific disabilities, new `HypertensionPdfGenerator` and `AsthmaPdfGenerator` classes that are subclasses of `FastTrackPdfGenerator` can be created. The new subclasses will probably have different implementations of `add_blood_pressure_outro`.

Reply:
@yoomlam how might this class delegate the type of rrd? Maybe it's handed the type-specific class

Reply:
You could pass the disability (`disability_symbol`) as an argument into the call to `new`.
Or create and use a method defined something like this:
```ruby
def single_disability_for(form526_submission)
  mapped_submission_disabilities = RapidReadyForDecision::Constants.extract_disability_symbol_list(form526_submission)
  disability_symbol = mapped_submission_disabilities.first
  disability_struct = RrdConstants::DISABILITIES[disability_symbol]
end
```
See https://github.com/department-of-veterans-affairs/vets-api/pull/9403#discussion_r833599970